src/main/java/mate/jdbc/dao/manufacturer/ManufacturerDaoImpl.java

```diff
@@ -18,7 +17,7 @@
 public class ManufacturerDaoImpl implements ManufacturerDao {
     @Override
     public Manufacturer create(Manufacturer manufacturer) {
-        String query = "INSERT INTO manufacturers (name, country) "
+        String query = "INSERT INTO manufacturer (name, country) "
```

Comment:
The table must be named in the plural
```suggestion
        String query = "INSERT INTO manufacturers (name, country) "
```